lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,14 +1,28 @@
 ;(function (define) {
     'use strict';
 
-    define(['backbone', 'underscore', 'text!teams/templates/teams-tab.underscore'],
-        function (Backbone, _, teamsTabTemplate) {
-            var TeamTabView = Backbone.View.extend({
-                render: function() {
-                    this.$el.html(_.template(teamsTabTemplate, {}));
-                }
-            });
+    define(['backbone',
+            'underscore',
+            'gettext',
+            'js/components/header/views/header',
+            'js/components/header/models/header',
+            'text!teams/templates/teams-tab.underscore'],
+           function (Backbone, _, gettext, HeaderView, HeaderModel, teamsTabTemplate) {
+               var headerModel = new HeaderModel({
```

Comment:
I agree that this makes sense. It might also protect against memory leaks, because you can reuse the header view rather than creating a new one each time.